### PR TITLE
fixed some todos

### PR DESCRIPTION
**Pull request by yevhenprudnik**
Hello @evgeniidatsiuk. I've fixed some todos that you signed me. Also I have some questions about few other todos/suggestions. You've recommended to create additional models (comment model, friend model, like model). Now 'like' and 'friends' fields of a post and user schemas are defined as arrays of strings, so I don't quite understand how should I use models for that fields. 
Could you please explain a bit further these comments:
"maybe would be better to create like model?)"
"maybe would be better to create comment model?)"
Thank you)

### CODE REVIEW

**Comment by evgeniidatsiuk** [comment]
Hi, @yevhenprudnik ;)

I wanted to say that it would be better to create Friend schema (model) that will help you to find your friends with pagination and sort by date and so on (like on Facebook). 

Why? Because in the future It will allow a lot of stuff todo, for example: you can notify users "You have been connected with Yevhenii 2 years" etc.

The same for comments and likes, it might be a different model (schema)

**Comment by evgeniidatsiuk** [comment]
Yes, it's a bit complicated, but it will be more professional than just an array )))


What do you think?

**Comment by yevhenprudnik** [comment]
Yes, I completely agree. That will be much better than usual array

File: models/user-model.js [hunk 1]
```diff
@@ -15,7 +14,7 @@ const UserSchema = new Schema({
   outcomingRequests: [String],
   incomingRequests: [String],
   avatar: {type: String, default: 'http://tachyons.io/img/avatar_1.jpg'},
-  createdVia: { type: String},
+  oauth: { type: Boolean},
```
Comment (evgeniidatsiuk) [inline models/user-model.js:17]: why? we need to know the source.

facebook/google/apple etc

**Review by evgeniidatsiuk — CHANGES_REQUESTED** [review]

File: models/user-model.js (hunk 1 above)
Comment (evgeniidatsiuk) [reply models/user-model.js:17]: also u missed facebook/google id

how can we get info facebook again?

Comment (yevhenprudnik) [reply models/user-model.js:17]: I don't understand what facebook/google id and what information do we need from facebook again?

Comment (evgeniidatsiuk) [reply models/user-model.js:17]: Email on facebook can be changed )

So we need to find user by facebook id, it is more reliable way

Comment (yevhenprudnik) [reply models/user-model.js:17]: okay, got it

**Pull request merged**